docroot/modules/custom/va_gov_backend/js/gtm_tag_trackers.es6.js

```diff
@@ -37,6 +37,9 @@
           ? settings.gtm_data.userRoles
           : null,
         userId: settings.gtm_data.userId ? settings.gtm_data.userId : null,
+        userSection: settings.gtm_data.userSection
```

Comment:
```suggestion
        userSections: settings.gtm_data.userSections
```

my bad here, missed the `s` in ACs